CHANGES.md

```diff
@@ -1,3 +1,18 @@
+For the next release
+====================
+
+Removal warning
+---------------
+
+Some older clients added `:` to the `client_secret` parameter of various endpoints, however the
+Client-Server API specification
+[disallows this](https://matrix.org/docs/spec/client_server/unstable#post-matrix-client-r0-register-email-requesttoken).
```

Comment:
This should probably link to the current latest spec, not unstable?

```suggestion
[disallows this](https://matrix.org/docs/spec/client_server/r0.6.1#post-matrix-client-r0-register-email-requesttoken).
```

Reply:
Oops, yep certainly should.